biothings/hub/dataload/dumper.py

```diff
@@ -483,10 +486,26 @@ class FTPDumper(BaseDumper):
     FTP_USER = ''
     FTP_PASSWD = ''
     FTP_TIMEOUT = 10 * 60.0  # we want dumper to timout if necessary
+    BLOCK_SIZE: Optional[int] = None  # default is still kept at 8KB
 
     # TODO: should we add a __del__ to make sure to close ftp connection ?
     # ftplib has a context __enter__, but we don't use it that way ("with ...")
 
+    def _get_optimal_buffer_size(self) -> int:
+        if self.BLOCK_SIZE is not None:
+            return self.BLOCK_SIZE
+        # else:
+        known_optimal_sizes = {
+            'ftp.ncbi.nlm.nih.gov': 33554432,
```

Comment:
@zcqian add the reference link here to explain why we set 32MB for ncbi ftp server?